BaragonUI/app/components/status/StatusPage.jsx

```diff
@@ -1,15 +1,37 @@
 import React from 'react';
 import { connect } from 'react-redux';
-import classNames from 'classnames';
-import { Link } from 'react-router';
+import { browserHistory } from 'react-router';
 import rootComponent from '../../rootComponent';
 import { refresh } from '../../actions/ui/status';
 
-const StatusPage = (props) => {
+import WorkerStatus from './WorkerStatus';
+import PendingRequests from './PendingRequests';
+import RequestSearch from './RequestSearch';
+
+const navigateToRequest = (requestId) => {
+  browserHistory.push(`/requests/${requestId}`);
```

Comment:
Thank you; I did think there had to be a better way to do this but I couldn't find it